mscore/palettebox.h

```diff
@@ -32,10 +34,13 @@ class PaletteBox : public QDockWidget {
       QVBoxLayout* vbox;
       Palette* newPalette(const QString& name, int slot);
       QComboBox* workspaceList;
-      QLineEdit* searchBox;
+
+      //QMap<QString, Shortcut*> localShortcuts;
+
       const int paletteStretch = 1000;
       QAction* singlePaletteAction;
       QToolButton* addWorkspaceButton;
+      bool keyboardNavigation = false;
```

Comment:
Currently, the behavior is more consistent - after applying an element, the highlighting is cleared, implying keyboard navigation really is disabled.  But maybe it would be better if it were consistent in the other direction - if keyboard navigation remained enabled.  So that returning to the search box put us right back where we were before, and could re-apply the same item again by hitting Enter again.